jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -56,8 +56,23 @@ YUI.add('inspector-component', function() {
       var state = {
         activeComponent: metadata.activeComponent
       };
+      var appPreviousState = nextProps.appPreviousState ||
+          this.props.appPreviousState;
+      var previousMetadata;
+      if (appPreviousState.hasOwnProperty('sectionA')) {
+        previousMetadata = appPreviousState.sectionA.metadata;
+      }
       switch (state.activeComponent) {
         case undefined:
+          var component;
```

Comment:
Can you add a comment here to what this is for? AIUI it's so that we can navigate back to a previous service..?